src/borg/remote.py

```diff
@@ -930,7 +949,62 @@ def preload(self, ids):
 
 
 def handle_remote_line(line):
-    if line.startswith('$LOG '):
+    """
+    Handle a remote log line.
+
+    This function is remarkably complex because it handles three different wire formats.
```

Comment:
s/three/multiple/

otherwise we'll forget to fix that when we remove one of them at 1.1rc.